src/init.hpp

```diff
@@ -124,6 +124,7 @@ namespace madrona_gpudrive
         bool disableClassicalObs = false;
         DynamicsModel dynamicsModel = DynamicsModel::Classic;
         bool readFromTracksToPredict = false;       // Default: false - for womd_tracks_to_predict initialization mode
+        uint32_t episode_length = 91;
```

Comment:
Seeming inconsistency here: episode_length = 91 here and max episode length is 200?